parse dsp file & save spectrum to csv

samo38 opened the pull request.

ehb54 commented:
@samo38 
Did you mean to lose the indent on https://github.com/ehb54/ultrascan3/pull/67/files#diff-7119ad4526b77ebc01cc805aaf9eee629ca2cb17a12f66d1deadf894429dfad8L839-R874 ?

Also indent looks wrong here https://github.com/ehb54/ultrascan3/pull/67/files#diff-5687c8561fd8fdb4f1d3cae0ab4572a536f75c428baf0190b41c5b3d3409a84eL85-R85

Personally, I don't like if() without a {} e.g. https://github.com/ehb54/ultrascan3/pull/67/files#diff-6b258d2b33df786bd7162cd48ed20a532ac93d2216b8f9d7391e3d31c71c0779R394-R398
But I guess it is ok.

samo38 commented:
I use QtCreator for coding. I didn't see any bad indentation of point one.
About the second point, I added a line to check to set the right numbers
but wasn't needed. Then I deleted it. The changes in the indentation come
from this. I'll fix this as well as add braces to the lines mentioned in
the third point.

On Tue, May 9, 2023 at 2:38 AM ehb54 ***@***.***> wrote:

> @samo38 <https://github.com/samo38>
> Did you mean to lose the indent on
> https://github.com/ehb54/ultrascan3/pull/67/files#diff-7119ad4526b77ebc01cc805aaf9eee629ca2cb17a12f66d1deadf894429dfad8L839-R874
> ?
>
> Also indent looks wrong here
> https://github.com/ehb54/ultrascan3/pull/67/files#diff-5687c8561fd8fdb4f1d3cae0ab4572a536f75c428baf0190b41c5b3d3409a84eL85-R85
>
> Personally, I don't like if() without a {} e.g.
> https://github.com/ehb54/ultrascan3/pull/67/files#diff-6b258d2b33df786bd7162cd48ed20a532ac93d2216b8f9d7391e3d31c71c0779R394-R398
> But I guess it is ok.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ehb54/ultrascan3/pull/67#issuecomment-1539193573>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALWX6SHU2WUU4NM3Q25UVSDXFH7ALANCNFSM6AAAAAAXWJPCSI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

samo38 commented:
Hi Emre. The indentations issues are fixed. I just checked that CodeQl runs
finished without errors on my new changes.
Thanks, -Saeed

On Tue, May 9, 2023 at 8:16 AM saeed mortezazadeh <
***@***.***> wrote:

> I use QtCreator for coding. I didn't see any bad indentation of point one.
> About the second point, I added a line to check to set the right numbers
> but wasn't needed. Then I deleted it. The changes in the indentation come
> from this. I'll fix this as well as add braces to the lines mentioned in
> the third point.
>
> On Tue, May 9, 2023 at 2:38 AM ehb54 ***@***.***> wrote:
>
>> @samo38 <https://github.com/samo38>
>> Did you mean to lose the indent on
>> https://github.com/ehb54/ultrascan3/pull/67/files#diff-7119ad4526b77ebc01cc805aaf9eee629ca2cb17a12f66d1deadf894429dfad8L839-R874
>> ?
>>
>> Also indent looks wrong here
>> https://github.com/ehb54/ultrascan3/pull/67/files#diff-5687c8561fd8fdb4f1d3cae0ab4572a536f75c428baf0190b41c5b3d3409a84eL85-R85
>>
>> Personally, I don't like if() without a {} e.g.
>> https://github.com/ehb54/ultrascan3/pull/67/files#diff-6b258d2b33df786bd7162cd48ed20a532ac93d2216b8f9d7391e3d31c71c0779R394-R398
>> But I guess it is ok.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/ehb54/ultrascan3/pull/67#issuecomment-1539193573>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ALWX6SHU2WUU4NM3Q25UVSDXFH7ALANCNFSM6AAAAAAXWJPCSI>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

ehb54 commented:
Thanks Saeed.
